web/css/toodles.css

```diff
@@ -53,6 +53,10 @@
     font-size: .75em;
 }
 
+.todo-source-link:hover {
+    font-weight: bold;    
```

Comment:
I definitely agree that adding some sort of visual indicator here is good. But this particular one is slightly problematic because it changes the size of the text, which then causes the entire table cell to grow and shrink when you hover and leave, which can be a bit jarring if you sweep the mouse down the screen. 

I think there's a few options here:

- Do a different kind of visual change eg color change, underlining, etc that don't alter the width of the text
- Do something clever with the table to simply prevent the width of that column from changing (I say this not having a concrete suggestion for how to do that)
- Revert the change, solve this some other way later.

Let me know what you think, happy to discuss more.